src/openrct2/localisation/string_ids.h

```diff
@@ -3748,105 +3748,112 @@ enum {
 
     STR_OBJECT_LOAD_ERROR_TITLE = 6123,
     STR_OBJECT_NAME = 6124,
-    STR_OBJECT_TYPE = 6125,
-    STR_UNKNOWN_OBJECT_TYPE = 6126,
-    STR_OBJECT_ERROR_WINDOW_FILE = 6127,
-    STR_OBJECT_ERROR_WINDOW_EXPLANATION = 6128,
-    STR_COPY_SELECTED = 6129,
-    STR_COPY_ALL = 6130,
-    STR_OBJECT_SOURCE = 6131,
+STR_OBJECT_TYPE = 6125,
```

Comment:
You've accidentally removed the indentation here. Could you please re-introduce it?